services/jex-events/Dockerfile

```diff
@@ -10,9 +10,6 @@ LABEL org.iplantc.de.jex-events.git-ref="$git_commit" \
       org.iplantc.de.jex-events.version="$version" \
       org.iplantc.de.buildenv.git-ref="$buildenv_git_commit"
 
-USER iplant
```

Comment:
There is a `useradd` command further up in this dockerfile which should be removed.